### ISSUE TITLE
8$: report missing expression in JSX prop

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#986]
```javascript
    <Line
      ref={line}
      x={-500}
      y={0}
      lineWidth={6}
      end={}
      points={[
          [100, 0],
          [0, 100],
      ]}
      stroke="#e13238"
    />
```

In this code, `end` is missing a value. This is illegal, and quick-lint-js should diagnose it.

**Comment by JPMoles** [comment]
Hi, I would like to work on this

**Comment by JPMoles** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.
